wp/resources/sample_binaries/memory_samples/mem_hooks/main_2.S

```diff
@@ -0,0 +1,23 @@
+.global _start
+
+.text
+
+main:
+    mov     (val),       %rax
+    nop
```

Comment:
Is this `nop` necessary? It didn't change the test-result when I removed it.

Reply:
Nope, not necessary. I just had it as padding to make the addresses different, but adding the `foo` symbol in the modified binary was sufficient enough. I can take this out!

Reply:
Fine either way. Actually maybe leaving it in to remind readers " `nop`'s can change addresses" is a good idea.